requirements.txt

```diff
@@ -14,3 +13,7 @@ smmap==0.9.0
 wheel==0.24.0
 PTable==0.9.2
 progressbar2==3.10.1
+pylint==1.8.1
+wrapt==1.10.11
+python-utils==2.2.0
+tqdm==4.19.5
```

Comment:
This was missing :|

Reply:
Ah dang, can you add a step in the release steps in `CONTRIBUTING.md` for updating `requirements.txt`?